src/components/Logo/index.tsx

```diff
@@ -1,22 +1,48 @@
 import { useState } from 'react'
-import { Slash } from 'react-feather'
 import { ImageProps } from 'rebass'
-import { useTheme } from 'styled-components/macro'
+import styled from 'styled-components/macro'
 
 const BAD_SRCS: { [tokenAddress: string]: true } = {}
 
 interface LogoProps extends Pick<ImageProps, 'style' | 'alt' | 'className'> {
   srcs: string[]
+  symbol?: string
+  size?: string
 }
 
+const getFontSize = (size?: string) => {
```

Comment:
Is there a less bespoke way to do this? Can the font size always be a quarter of the size to standardize this?